manifests/install/charts/as-a-second-scheduler/values.yaml

```diff
@@ -5,31 +5,20 @@
 
 scheduler:
   name: scheduler-plugins-scheduler
-  image: k8s.gcr.io/scheduler-plugins/kube-scheduler:v0.22.6
+  image: k8s.gcr.io/scheduler-plugins/kube-scheduler:v0.23.10
   namespace: scheduler-plugins
   replicaCount: 1
+  leaderElect: false
 
 controller:
   name: scheduler-plugins-controller
-  image: k8s.gcr.io/scheduler-plugins/controller:v0.22.6
+  image: k8s.gcr.io/scheduler-plugins/controller:v0.23.10
   namespace: scheduler-plugins
   replicaCount: 1
 
 # LoadVariationRiskBalancing and TargetLoadPacking are not enabled by default
 # as they need extra RBAC privileges on metrics.k8s.io.
 
 plugins:
-  enabled: ["Coscheduling","CapacityScheduling","NodeResourceTopologyMatch", "NodeResourcesAllocatable"]
-
-global:  
-   # queueSort is not indented under extensions 
-   # as it needs to be globally enabled once. 
-  queueSort: ["Coscheduling"]
-  extensions: 
```

Comment:
@helayoty FYI: with the latest v1beta3 config, we can simplify the Helm parameters by just specifying which plugins should be enabled/disabled.

Reply:
much better :)